ha/util/conf_store.py

```diff
@@ -221,3 +221,19 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_node_id(index):
+        """
+        Get machine_id for control-node pod.
+        """
+        node_id = ''
+        try:
+            node_key = Conf.search(index, GconfKeys.NODE_CONST.value,
+                                   GconfKeys.TYPE_CONST.value, GconfKeys.CONTROL_NODE.value)
```

Comment:
Does it required , Conf.load ?? can u please check .

Reply:
This is a ConfstoreSearch class, no Conf.load is required.
The func is taking index as a parameter, so from wherever this func call will be made, there Conf.load will be done and the index will be parsed.

Reply:
ok thanks